resources/lib/monitor.py

```diff
@@ -65,9 +65,11 @@ def onNotification(self, sender, method, data):
             data = json.loads(data)[0]
 
         elif sender.startswith('upnextprovider'):
-            method = method.split('.')[1]
+            LOG.info('Attempting to play the next episode via upnext')
+            method = '.'.join(method.split('.')[1:])
```

Comment:
```suggestion
            method = method.split('.', 1)[1]
```
this is slightly faster :)